admin-ui/app/dialogs/send-push-notification.html

```diff
@@ -14,7 +14,16 @@ <h4 class="modal-title" id="myModalLabel">Send push to Application One</h4>
       </div>
     </div>
 
-
+      
```

Comment:
OK, sounds reasonable

On Thu, Mar 24, 2016 at 3:42 PM, Hoyt Summers Pittman <
notifications@github.com> wrote:

> In admin-ui/app/dialogs/send-push-notification.html
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/701#discussion_r57326586>
> :
>
> > @@ -14,7 +14,16 @@ <h4 class="modal-title" id="myModalLabel">Send push to Application One</h4>
> >        </div>
> >      </div>
> >
> > -
> > +
>
> High wakes the device and your app up. Normal messages are only processed
> when the device wakes up for regular work.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/701/files/b495835292a5ccfaeae32c45743a21d2dd49f9cd#r57326586>
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf